resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentCredentialServiceImpl.java

```diff
@@ -0,0 +1,275 @@
+							ResponseWrapper.class, tokenGenerator.getToken());
+					residentCredentialResponseDto = JsonUtil
+							.readValue(JsonUtil.writeValueAsString(responseDto.getResponse()), ResidentCredentialResponseDto.class);
+				
+						//sendNotification(dto.getIndividualId(), IdType.valueOf("UIN"),
+							//	NotificationTemplateCode.RS_CREDENTIAL_SUCCESS, null);
+			   } else {
+				logger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.APPLICATIONID.toString(),
+						LoggerFileConstant.APPLICATIONID.toString(),
+						ResidentErrorCode.OTP_VALIDATION_FAILED.getErrorMessage());
+			//	sendNotification(dto.getIndividualId(), IdType.valueOf("UIN"),
+				//		NotificationTemplateCode.RS_CREDENTIAL_FAILURE, null);
+				//throw new ResidentServiceException(ResidentErrorCode.OTP_VALIDATION_FAILED.getErrorCode(),
+					//	ResidentErrorCode.OTP_VALIDATION_FAILED.getErrorMessage());
+			}
+		}/* catch (OtpValidationFailedException e) {
+			sendNotification(dto.getIndividualId(), IdType.valueOf("UIN"),
+					NotificationTemplateCode.RS_CREDENTIAL_FAILURE, null);
+			throw new ResidentServiceException(ResidentErrorCode.OTP_VALIDATION_FAILED.getErrorCode(),
+					e.getErrorText(), e);
+		}*/
+	/*	catch (ResidentServiceCheckedException e) {	
+			//sendNotification(dto.getIndividualId(), IdType.valueOf("UIN"),
+					//NotificationTemplateCode.RS_CREDENTIAL_FAILURE, null);
+			throw new ResidentServiceException(ResidentErrorCode.API_RESOURCE_ACCESS_EXCEPTION.getErrorCode(),
+					ResidentErrorCode.API_RESOURCE_ACCESS_EXCEPTION.getErrorMessage(), e);
+		}*/
+		catch (ApisResourceAccessException e) {
+		//	sendNotification(dto.getIndividualId(), IdType.valueOf("UIN"),
+			//		NotificationTemplateCode.RS_CREDENTIAL_FAILURE, null);
+			throw new ResidentServiceException(ResidentErrorCode.API_RESOURCE_ACCESS_EXCEPTION.getErrorCode(),
+					ResidentErrorCode.API_RESOURCE_ACCESS_EXCEPTION.getErrorMessage(), e);
+		}
+		catch (IOException e) {
+			//sendNotification(dto.getIndividualId(), IdType.valueOf("RID"),
+				//	NotificationTemplateCode.RS_CREDENTIAL_FAILURE, null);
+			throw new ResidentServiceException(ResidentErrorCode.IO_EXCEPTION.getErrorCode(),
+					ResidentErrorCode.IO_EXCEPTION.getErrorMessage(), e);
+		}
+
+		return residentCredentialResponseDto;
+	}
+	
+	@Override
+	public CredentialRequestStatusResponseDto getStatus(String requestId)
+			throws ResidentServiceCheckedException {
+		ResponseWrapper<CredentialRequestStatusResponseDto> responseDto = null;
+		CredentialRequestStatusResponseDto credentialRequestStatusResponseDto=new CredentialRequestStatusResponseDto();
+		try {
+			String credentialUrl=credentailStatusUrl+requestId;
+			URI credentailStatusUri = URI.create(credentialUrl);
+			responseDto =residentServiceRestClient.getApi(credentailStatusUri, ResponseWrapper.class, tokenGenerator.getToken());
+			credentialRequestStatusResponseDto = JsonUtil
```

Comment:
get status also need to sentnotification based id it received from response.
Same for cancel